tracee-ebpf/main.go

```diff
@@ -872,6 +877,38 @@ func checkRequiredCapabilities() bool {
 	return caps.Get(capability.EFFECTIVE, capability.CAP_SYS_ADMIN)
 }
 
+func missingKernelConfigOptions() []string {
+
+	requiredConfigOptions := map[uint32]string{
+		helpers.CONFIG_BPF:               "y",
+		helpers.CONFIG_BPF_SYSCALL:       "y",
+		helpers.CONFIG_KPROBE_EVENTS:     "y",
+		helpers.CONFIG_BPF_EVENTS:        "y",
+		helpers.CONFIG_TRACING:           "y",
+		helpers.CONFIG_FTRACE_SYSCALLS:   "y",
+		helpers.CONFIG_HAVE_EBPF_JIT:     "y",
+		helpers.CONFIG_BPF_JIT:           "y",
+		helpers.CONFIG_BPF_JIT_ALWAYS_ON: "y",
+	}
+
+	kConfig := helpers.KernelConfig{}
+	helpers.InitKernelConfig(kConfig)
```

Comment:
Shouldn't you use the returned kernelconfig value and check for error like you do in populateBPFMaps()?

Reply:
Sloppy by me not catching this when I refactored. InitKernelConfig doesn't need to return the KernelConfig, no reason for this not to be a method.